Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIStorageCocoa.mm

```diff
@@ -75,6 +79,21 @@
     return *m_sync;
 }
 
+WebExtensionAPIStorageArea& WebExtensionAPIStorage::storageAreaForType(WebExtensionStorageType storageType)
+{
+    switch (storageType) {
+    case WebExtensionStorageType::Local:
+        return local();
+    case WebExtensionStorageType::Session:
+        return session();
+    case WebExtensionStorageType::Sync:
+        return sync();
+    }
+
+    ASSERT_NOT_REACHED();
+    return local();
```

Comment:
not sure what would be best to return here if this is reached.

Reply:
This seems fine.